SDK-1737: New Spryker Code Upgrader documentation (#1628)
spryker/spryker-docs: docs/paas-plus/dev/invite-users-to-spryker-ci.md

---
title: Invite users to Spryker CI
description: Instructions for inviting users to Spryker CI
template: howto-guide-template
---

> To collaborate with a team mate you need to invite them to the workspace and add them to the selected project. The exceptions are admins and the workspace owner who have access to all projects in the workspace.

Reviewer: ```suggestion
To collaborate with a teammate you need to invite them to your workspace and add them to the selected project. The only exceptions are admins and the workspace owner who have access to all projects in the workspace.
```

> To invite the teammate, do the following:

Reviewer: ```suggestion
To invite a teammate, do the following:
```
